cloudigrade/account/management/commands/add_account.py

```diff
@@ -62,6 +67,75 @@ def _do_saves(self, account, instances_data):
                 saved_instances[region].append(instance)
         return dict(saved_instances)
 
+    def _do_thing(self):
+        """Do nothing of value."""
```

Comment:
@infinitewarp Don't be so hard on yourself, you can add lots of value with your methods. 😉